crates/uv/tests/it/sync.rs

```diff
@@ -10986,6 +10986,24 @@ fn transitive_group_conflicts_shallow() -> Result<()> {
         "#,
     )?;
 
+    uv_snapshot!(context.filters(), context.lock(), @r"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Resolved 5 packages in [TIME]
+    ");
+
+    uv_snapshot!(context.filters(), context.lock().arg("--check"), @r"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Resolved 5 packages in [TIME]
+    ");
```

Comment:
You can see the failure from the issue in https://github.com/astral-sh/uv/pull/15884/commits/44522f0d0b9e71cc35d5b566d14c5ebd986e1b23